[1.1.1.1 Setup Documentation] adds KDE Plasma instructions in linux.md (#3891)
cloudflare/cloudflare-docs: content/1.1.1.1/setup/linux.md

6. Then, go to **IPv6**.
7. {{<render file="_all-ipv6.md">}}
8. Click **Apply**.
  
## KDE Plasma (Kubuntu, KDE neon, Fedora KDE, etc)
  
1. Go to **System Settings** > **Connections**.
2. Select the connection you want to configure - like your current connected network.
3. Click the **IPv4** tab.

> 4. Click the **Method** section, which will open a drop down menu - select the **Automatic (Only addresses)** option.

Reviewer: ```suggestion
4. Click the **Method** drop-down menu, and select _Automatic (Only addresses)_.
```

> 5. Select the text box next to **DNS servers**.

Reviewer: ```suggestion
5. Click the text box next to **DNS servers**.
```

6. {{<render file="_all-ipv4.md">}}
7. Click the **IPv6** tab.

> 8. Click the **Method** section, which will open a drop down menu - select the **Automatic (Only addresses)** option.
> 9. Select the text box next to **DNS servers**.

Reviewer: ```suggestion
8. Click the **Method** drop-down menu, and select _Automatic (Only addresses)_.
9. Click the text box next to **DNS servers**.
```
